index.md

```diff
@@ -1,5 +1,5 @@
 ---
-title: Flutter is a mobile app SDK for iOS and Android
+title: Welcome to Flutter
```

Comment:
Good - feel free to do that in a separate PR...I'm not comfortable as it's not standard industry practice in documentation, to let the SEO tail wag doc titles. A technical writer prioritizes usability and recognition for new users.